noxfiles/ci_nox.py

```diff
@@ -125,16 +125,20 @@ def fides_db_scan(session: nox.Session) -> None:
     """Scan the fides application database to check for dataset discrepancies."""
     session.notify("teardown")
     session.run(*START_APP, external=True)
-    run_command = (
-        *RUN,
+    scan_command = (
+        "docker",
+        "container",
+        "exec",
+        "fides-fides-1",
```

Comment:
Should this use the constant added in `constants_nox.py`?

```suggestion
        CONTAINER_NAME,
```

Reply:
good catch!